src/server/pps/server/monitor.go

```diff
@@ -456,7 +456,7 @@ func (m *ppsMaster) makeCronCommits(ctx context.Context, in *pps.Input) error {
 		}
 		if in.Cron.Overwrite {
 			// get rid of any files, so the new file "overwrites" previous runs
-			err = pachClient.DeleteFile(client.NewCommit(in.Cron.Repo, "master", ""), "")
+			err = pachClient.DeleteFile(client.NewCommit(in.Cron.Repo, "master", ""), "/")
```

Comment:
:+1:

Reply:
I am thinking about making this more structured so we don't need to depend on the trailing slash.